### PR TITLE
hello-world: update to new x-common canonical-data

### PR DESCRIPTION
**Pull request by robphoenix**
see https://github.com/exercism/x-common/pull/544

### CODE REVIEW

File: exercises/hello-world/hello-world.lua [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+-- Table to be returned by the hello-world module.
+local hello_world = {}
+
+-- Add the hello() function to the table returned by this module.
+function hello_world.hello()
+  -- Write some code here to pass the test suite.
+
+  -- When you have a working solution, REMOVE ALL THE STOCK COMMENTS.
+  -- They're here to help you get started but they only clutter a finished solution.
+  -- If you leave them in, reviewers will protest!
```
Comment (ryanplusplus) [inline exercises/hello-world/hello-world.lua]: :+1:

**Comment by ryanplusplus** [comment]
Perfect, thanks!

**Comment by robphoenix** [comment]
No worries.  :+1:
btw I'm happy to merge my PR's once they've been approved if that's easier? No worries either way. :smile:

**Comment by ryanplusplus** [comment]
No big deal for me to merge them. I'm just glad to have someone else to take a second look at everything.

**Pull request merged**